components/schedule/ScheduleBlock.vue

```diff
@@ -46,6 +47,7 @@ export default {
                 '-primary': this.primary,
                 '-secondary': this.secondary,
                 '-tertiary': this.tertiary,
+                '-tab-header': this.tabHeader,
                 '-active': this.active,
                 '-mini': this.mini,
                 '-transparent': this.transparent,
```

Comment:
Should it be `'cursor-pointer': this.tabHeader,` ?

Reply:
Means line 55.

Reply:
nice catch!
should be true when `this.primary` or `this.tabHeader` is true.